resources/assets/styles/components/_swiper.scss

```diff
@@ -1,28 +1,137 @@
 /* stylelint-disable block-no-empty */
+/* stylelint-disable no-descending-specificity */
```

Comment:
If finding yourself having to use this, it usually that means the file should be refactored.

Reply:
this happens because I want shared-styles for the next and previous (without calling out the same styles in each instance) and then because I need the specific 1-2 line diff between the 2 arrows (for positioning)